litestar/types/serialization.py

```diff
@@ -57,3 +62,5 @@
 EncodableMsgSpecType: TypeAlias = "Ext | Raw | Struct"
 LitestarEncodableType: TypeAlias = "EncodableBuiltinType | EncodableBuiltinCollectionType | EncodableStdLibType | EncodableStdLibIPType | EncodableMsgSpecType | BaseModel | AttrsInstance"  # pyright: ignore
 DataContainerType: TypeAlias = "Struct | BaseModel | AttrsInstance | TypedDictClass | DataclassProtocol"  # pyright: ignore
+PydanticV2FieldsListType: TypeAlias = "Set[int] | Set[str] | Dict[int, Any] | Dict[str, Any] | None"
```

Comment:
just for my understanding, is it ok to have ` | None` in the type alias? I would prefer `PydanticV2FieldsListType | None` just to be explicit, but I am not sure if that is the norm 🙂